letsencrypt/client/tests/crypto_util_test.py

```diff
@@ -63,9 +62,9 @@ class MakeKeyTest(unittest.TestCase):  # pylint: disable=too-few-public-methods
 
     def test_it(self):  # pylint: disable=no-self-use
         from letsencrypt.client.crypto_util import make_key
+        # This individual test was taking over 6 seconds...
+        # I have shortened it... to aid debugging the rest of the project
         M2Crypto.RSA.load_key_string(make_key(1024))
-        M2Crypto.RSA.load_key_string(make_key(2048))
-        M2Crypto.RSA.load_key_string(make_key(4096))
```

Comment:
so you want to only test with the key length noone will use in practice for the sake of test speed?